src/lib/validatePassword.js

```diff
@@ -0,0 +1,7 @@
+import assertString from './util/assertString';
+
+export default function validatePassword(str) {
+  const format = /[ !@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]/;
+  assertString(str);
+  return (/[a-z]/.test(str)) && (/[A-Z]/.test(str)) && (/[\d]/.test(str)) && format.test(str);
```

Comment:
This looks good.

Few suggestions:
- Perhaps we `minLength` parameter?
- Can rename this validator to `isStrongPassword`